ui/apps/platform/src/Containers/Collections/RuleSelector/ByNameSelector.tsx

```diff
@@ -77,6 +93,8 @@ function ByNameSelector({
                                     : ValidatedOptions.default
                             }
                             isDisabled={isDisabled}
+                            autocompleteProvider={onAutocomplete}
```

Comment:
A question for understanding, not necessarily a request for change. The difference between prop name and prop value function name.
* Leftover from initial draft of code?
* Intended to communicate meaning to reader?

Reply:
A little bit of both, but probably more of the former. This was something in the back of my head but since you called it out I think it is worth fixing now. I'll update all to `autocompleteProvider`, since `onAutocomplete` implies the function is a callback, and this is being passed to a declarative data fetching API.